.github/workflows/pull-request-target.yml

```
@@ -19,26 +19,31 @@ jobs:
     steps:
       - name: Add the PR Review Policy
-        uses: thollander/actions-comment-pull-request@v2
+        uses: thollander/actions-comment-pull-request@v3
         with:
-          comment_tag: pr_review_policy
+          comment-tag: pr_review_policy
           message: |
             ## Our Pull Request Approval Process
 
-            We have these basic policies to make the approval process smoother for our volunteer team.
+            Thanks for contributing! 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)